pkg/metrics/constants.go

```diff
@@ -18,7 +18,7 @@ package metrics
 
 const (
 	// Namespace(s).
-	Namespace = "karpenter"
+	Namespace = "kube-system"
```

Comment:
We don't want ot change this. This is the prometheus concept of namespace not kubernetes. 

See: https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Opts.Namespace 


it is used as the prefix to our metrics